### PR TITLE
Update README.zh-CN.md

### PR DESCRIPTION
**Pull request by byronhe**

### CODE REVIEW

File: README.zh-CN.md
Comment (Niennienzz) [inline README.zh-CN.md:122]: 建议在末尾增加默认值说明 `默认为 false。`

**Review by Niennienzz — COMMENTED** [review]
Thanks for contributing! Left a minor suggestion.

File: README.zh-CN.md
Comment (byronhe) [reply README.zh-CN.md]: done

**Review by Niennienzz — APPROVED** [review]
LGTM

@romange needs approval by reviewers with write access 👍

**Pull request merged**
